nodes/common/pulp_node/error.py

```diff
@@ -1,5 +1,13 @@
 from gettext import gettext as _
 
+CLI_DEPRECATION_WARNING = \
```

Comment:
Yes that is what you are looking for. So `pulp-nodes-common` looks like it's required from both `pulp-admin` and `pulp-consumer` which is good. This looks like it will be a non-issue, thank you for checking.